src/edgeProxy/public/javascripts/uio.js

```diff
@@ -1,19 +1,163 @@
+/* global getCookieValue */
+
 "use strict";
 
-fluid.uiOptions(".flc-prefsEditor-separatedPanel", {
-    auxiliarySchema: {
-        terms: {
-            "templatePrefix": "lib/infusion/src/framework/preferences/html",
-            "messagePrefix": "lib/infusion/src/framework/preferences/messages"
-        },
-        "fluid.prefs.tableOfContents": {
-            enactor: {
-                "tocTemplate": "lib/infusion/src/components/tableOfContents/html/TableOfContents.html",
-                "tocMessage": "lib/infusion/src/framework/preferences/messages/tableOfContents-enactor.json",
-                ignoreForToC: {
-                    "overviewPanel": ".flc-overviewPanel"
-                }
+fluid.registerNamespace("fluid.prefs.edgeProxy");
+
+fluid.defaults("fluid.dataSource.noEncoding", {
+    gradeNames: ["fluid.dataSource", "fluid.dataSource.writable"],
+    components: {
+        encoding: {
+            type: "fluid.dataSource.encoding.none"
+        }
+    }
+});
```

Comment:
Not sure if this encapsulation of config into a grade is necessary, as it only seems to be used by `fluid.prefs.edgeProxystore`.